### ISSUE TITLE
for some reason they moved the url

### ISSUE THREAD

**Issue by CF-Cat-Boy** [AstroTechies/astro_modloader#22]
they moved "astroneermods.space" to "astromod.space"

**Comment by Famous5000** [comment]
yeah uh I was told that the new site is apparently in violation of their copyright. So... I don't think the modloader is ever planned to regain functionality for auto-updates. Which sucks but oh well...

**Comment by CF-Cat-Boy** [comment]
sad

**Comment by RedstoneWizard08** [comment]
@Famous5000 @CF-Cat-Boy the site was actually taken down by its owner because they no longer care about the game. As far as I know, astromod.space is unofficial and potentially unsafe. An archive can be found at https://atenfyr.com/ams-archive

I'd recommend joining the discord: https://discord.gg/bBqdVYxu4k

P.S. I'm working on an alternative place to host the mods since all this happened! :D

**Comment by RedstoneWizard08** [comment]
> I don't think the modloader is ever planned to regain functionality for auto-updates. Which sucks but oh well...

If my system gets used for the new place for mods, I'll make sure of it, as well as a URL protocol to install mods with a single click. (And maybe a new UI that supports browsing mods? Idk. Maybe not for that one.)
